ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -373,6 +381,19 @@ VAV_PDSCV_CO2C_VOCPC_ZHM_RMM:
   - ZHM
   - RMM
 
+VAV_SD_DSP_HWDC_MWDPC_CO2C_ZHM2X:
+  guid: "db3da40b-cc37-4466-b182-7ed5a048b632"
+  description: "Single duct reheat VAV with CO2, temp and differential pressure control and zone humidity monitoring."
+  is_canonical: true
+  implements:
+  - VAV
+  - SD
+  - DSP
+  - HWDC
+  - MWDPC
```

Comment:
Minimum water DP??

Reply:
MWDPC does not have any water specific points and this vav uses a minimum DP control. WDPC and WDPC2X description says "Differential pressure control in whichever system." We think it is best to change MWPDC description to "Minimum differential pressure control in whichever system." instead of creating a new type MPDC which would have the same points as MWDPC.

Reply:
After discussion, this will be investigated further to understand pressure sensor and setpoint type.